test/sanity/sanity.go

```diff
@@ -0,0 +1,36 @@
+package sanity
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+
+	"github.com/ghodss/yaml"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+	"github.com/openshift/openshift-azure/test/e2e/standard"
+	testlogger "github.com/openshift/openshift-azure/test/util/log"
+)
+
+// Checker is a singleton sanity checker which is used by all the Ginkgo tests
+var Checker *standard.SanityChecker
+
+func init() {
+	b, err := ioutil.ReadFile("../../_data/containerservice.yaml") // running via `go test`
```

Comment:
Is this how we replace all the dir walker?

Reply:
yes